src/init.cpp

```diff
@@ -475,6 +475,7 @@ std::string HelpMessage(HelpMessageMode mode)
         strUsage += HelpMessageOpt("-incrementalrelayfee=<amt>", strprintf("Fee rate (in %s/kB) used to define cost of relay, used for mempool limiting and BIP 125 replacement. (default: %s)", CURRENCY_UNIT, FormatMoney(DEFAULT_INCREMENTAL_RELAY_FEE)));
         strUsage += HelpMessageOpt("-dustrelayfee=<amt>", strprintf("Fee rate (in %s/kB) used to defined dust, the value of an output such that it will cost about 1/3 of its value in fees at this fee rate to spend it. (default: %s)", CURRENCY_UNIT, FormatMoney(DUST_RELAY_TX_FEE)));
     }
+    strUsage += HelpMessageOpt("-dustlimit=<amt>", strprintf(_("Amount of DOGE under which a transaction output is considered dust, in Koinu (default: %u)"), DEFAULT_DUST_LIMIT));
```

Comment:
From what I understand we're expressing this in COIN, not Koinu; so a fraction of COIN, not whole Koinu.